databricks_asset_bundles/03.evaluate_model.py

```
@@ -0,0 +1,188 @@
+# COMMAND ----------
+
+if mae_new < mae_old:
+    print("New model is better based on MAE.")
+    model_version = mlflow.register_model(
+      model_uri=new_model_uri,
+      name=f"{catalog_name}.{schema_name}.nyctaxi_model_fe",
+      tags={"git_sha": f"{git_sha}",
+            "job_run_id": job_run_id})
+
+    print("New model registered with version:", model_version.version)
+    dbutils.jobs.taskValues.set(key="model_version", value=model_version.version)
```

Ruff (0.7.0)

Undefined name `dbutils`

(F821)